### PR TITLE
Group Owners in Braid

### PR DESCRIPTION
**Pull request by hellobart**
- added teams config
- clicking on members switch users
- other options are hidden in settings

Demo: https://cl-braid.surge.sh/

### CODE REVIEW

File: src/teams.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+export const teams = [
+  {
+    name: 'Core',
+    membersId: [3244710, 3331816, 1114192, 3314768] // Spook, Ryan, Steve, Sam
```
Comment (rupurt) [inline src/teams.js:4]: How do you feel about pulling out each `membersId` into a separate variable so that it's self documenting?

Comment (hellobart) [reply src/teams.js]: Done

File: src/teams.js [hunk 2]
```diff
@@ -1,22 +1,38 @@
+const SPOOK = 3244710;
+const RYAN = 3331816;
+const Steve = 1114192;
```
Comment (rupurt) [inline src/teams.js]: How do you feel about `STEVE` to keep it consistent?

Comment (hellobart) [reply src/teams.js]: Hey, This was fixed right after second commit :)

**Pull request merged**